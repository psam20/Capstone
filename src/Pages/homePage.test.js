import React from 'react';
import {shallow} from 'enzyme';
import HomePage from './homepage';

describe('Check If HomePage Components Renders Correctly',()=>{
    let Wrapper=shallow(<HomePage />);
    it('Should Render HomePage Component Correctly',()=>{
        expect(Wrapper).toMatchSnapshot();
    })
})