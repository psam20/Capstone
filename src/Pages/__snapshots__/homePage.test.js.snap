// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check If HomePage Components Renders Correctly Should Render HomePage Component Correctly 1`] = `ShallowWrapper {}`;
